.github/workflows/coverage.yml

```diff
@@ -78,7 +78,7 @@ jobs:
         if: "env.GIT_DIFF != ''"
       - name: test & coverage report creation
         run: |
-          cat xab.txt | xargs go test -mod=readonly -timeout 5m -race -coverprofile=coverage.txt -covermode=atomic
+          cat xab.txt | xargs go test -mod=readonly -timeout 8m -race -coverprofile=coverage.txt -covermode=atomic
```

Comment:
Did Ci slow down? What is the reason for this change?

Reply:
yes, mostly likely caused by `TestDontExhaustMaxActiveIDs` as it's already time-consuming and now testing two difference mempools may just double its time consumption